nav2_dijkstra_planner/README.md

```diff
@@ -0,0 +1,43 @@
+# Dijkstra Planner
+The DijkstraPlanner is an implementation of a [Planning module](../doc/requirements/requirements.md) (equivalent to GlobalPlanner in ROS1 [MoveBase](http://wiki.ros.org/move_base)).
+
+The Planning module is responsible for generating a feasible path given start and end robot poses.
+
+## Status
+Currently, DijkstraPlanner is a direct port from ROS1 MoveBase [Navfn](http://wiki.ros.org/navfn) planner.
+
+The Navfn planner assumes a circular robot and operates on a costmap to find a minimum cost plan from a start point to an end point in a grid. The navigation function is computed with Dijkstra's algorithm. The strategy is based on [High-Speed Navigation Using
+the Global Dynamic Window Approach](https://cs.stanford.edu/group/manips/publications/pdfs/Brock_1999_ICRA.pdf) Brock, O. and Oussama K. IEEE (1999)
+
+## Task Interface
+
+The [Navigation System]((../doc/requirements/requirements.md)) is composed of three tasks: NavigateToPose, ComputePathToPose and FollowPathToPose.
+
+The DijkstraPlanner implements the Task Server interface for ComputePathToPose, specifically derives from `ComputePathToPoseTaskServer`. The client to DijkstraPlanner is 'NavigateToPoseTask', which periodically sends requests to the path planner.
+
+![alt text](../doc/design/NavigationSystemTasks.png "Navigation Tasks")
+
+## Testing
+Currenty, component level tests are available that exercise the Planning Module Task Server API. The intention is to develop standard tests that can be used for any type of planner.
+
+A PlannerTester node provides the world representation in the form of a costmap, sends a request to generate a path, and receives and checks the quality of the generated path.
+
```

Comment:
I'm not sure that the testing explanation should overshadow the actual package's description. To be fair, its a really neat picture you generated, but I might just follow the KISS principle here. Probably involving removing all the text (lines 25-33) below this comment

Reply:
Agree, this explanation should be removed from this package description and moved to system tests.

Reply:
Or even the tests as well moved over there